### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by matavak**
Added project Description/ objective / core features
formatted the README file
added the technologies to be used in development

### CODE REVIEW

**Review by CHADBUCK13 — CHANGES_REQUESTED** [review]

File: README.md [hunk 1]
```diff
@@ -1,14 +1,49 @@
...
+- Cancel an order
 
 
+ ## Team Members:
+ 
+- Chad Buck - CHADBUCK13
+- Dany Khaled - danikh25
+- Aida Berhe - aberhenegousse
+- Matin Tavakkoli- matavak
+- Henry zhixing Li - 40091330
```
[the remark this answers is not in the document]
Comment (matavak) [reply README.md]: changed id to git name

File: README.md [hunk 2]
```diff
@@ -1,14 +1,49 @@
...
+- Mathieu Trudeau - mathieutrudeau
+- Hawa Diallo - hawadi
+
+## Technologies:
+### Front end:
+- Django
+- CSS
+- HTML
+### Back end:
+- MySQL/PHP
```
[the remark this answers is not in the document]
Comment (matavak) [reply README.md]: chaanged to firebase

**Review by matavak — COMMENTED** [review]
added the changes to ID and backend features

**Review by mathieutrudeau — APPROVED** [review]
Looks good.

File: README.md [hunk 3]
```diff
@@ -1,14 +1,49 @@
...
+- Place an order
+- Pay for the order
+- Cancel an order
 
 
+ ## Team Members:
+ 
+- Chad Buck - CHADBUCK13
+- Dany Khaled - danikh25
+- Aida Berhe - aberhenegousse
```
Comment (CHADBUCK13) [inline README.md:33]: Change this name please: Aida Berhe-Negousse

**Review by CHADBUCK13 — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (matavak) [reply README.md]: changed

File: README.md [hunk 4]
```diff
@@ -1,14 +1,49 @@
-# soen341
-Project for SOEN 341
+# soen341: Software Process
+Winter 2022 group 9  -Dr. Tse-Hsun (Peter) Chen
+## Introduction:
```
Comment (CHADBUCK13) [inline README.md:3]: Remove this

File: README.md [hunk 5]
```diff
@@ -1,14 +1,49 @@
...
+### Objective: 
+develop an e-commerce website using agile methodologies.
```
Comment (CHADBUCK13) [inline README.md:5]: Capitalized the D in develop

File: README.md [hunk 6]
```diff
@@ -1,14 +1,49 @@
...
+### Description:
+An e-commerce website allows users to buy and sell goods and services over the internet rather than in person.
+using this website, a merchant can process orders, accept payments and manage shipping to customers.
```
Comment (CHADBUCK13) [inline README.md:8]: Capitalize U

File: README.md [hunk 7]
```diff
@@ -1,14 +1,49 @@
...
+there are two types of users for this website:
+   - Customer
+   - Seller
+   
+these users will be able to choose which kind they are when signing up for the website.
+A seller is able to:
+- Add items for sale 
+- Upload pictures for the items
 
+A user is able to:
```
Comment (CHADBUCK13) [inline README.md:19]: Change user to buyer

File: README.md [hunk 8]
```diff
@@ -1,14 +1,49 @@
...
+using this website, a merchant can process orders, accept payments and manage shipping to customers.
+### Core features:
+there are two types of users for this website:
+   - Customer
+   - Seller
+   
+these users will be able to choose which kind they are when signing up for the website.
+A seller is able to:
+- Add items for sale 
+- Upload pictures for the items
```
Comment (CHADBUCK13) [inline README.md:17]: They can also add description, prices, sales and categories

File: README.md [hunk 9]
```diff
@@ -1,14 +1,49 @@
...
+- Henry zhixing Li - JoliopoLi 
+- Mathieu Trudeau - mathieutrudeau
+- Hawa Diallo - hawadi
+
+## Technologies:
+### Front end:
+- Django
+- CSS
+- HTML
+### Back end:
```
Comment (CHADBUCK13) [inline README.md:44]: Firebase for users, they can use authorization services such as Google, Facebook and Apple

**Review by CHADBUCK13 — CHANGES_REQUESTED** [review]

File: README.md [hunk 10]
```diff
@@ -1,14 +1,49 @@
...
+- Place an order
+- Pay for the order
+- Cancel an order
 
 
+ ## Team Members:
+ 
+- Chad Buck - CHADBUCK13
+- Dany Khaled - danikh25
+- Aida Berhe- Negousse
```
Comment (aberhenegousse) [inline README.md:33]: Aida Berhe-Negousse - aberhenegousse

**Review by aberhenegousse — APPROVED** [review]

**Review by CHADBUCK13 — APPROVED** [review]

**Pull request merged**